src/fn_arg.hh

```diff
@@ -144,6 +149,21 @@ class Base {
  public:
     virtual void init_multi_ys() = 0;
     virtual const Yield::Multi &multi_ys() const { return m_ys; }
+
+    const bool is_partof_outside() {
```

Comment:
Shouldn't this be `bool is_partof_outside() const {`? 

Returning `const bool` would indicate that the boolean returned from this function is constant, i.e. doesn't change. Since there is a `set_partof_outside` method though, I would assume that this boolean can be changed at runtime, which I think can be confusing if there is a `const bool` method for the same boolean.  `bool is_partof_outside() const`, however, just means that this method doesn't change the internal state of the object is belongs to, which applies here.

Reply:
Yes, that's what I thought. It's quite easy to accidentally put the `const` in the wrong place. I found the same issue in `grammar.hh` (see below), so maybe this will also appear in some other files, in which case I won't point it out again.